runtime_builders/builder_util.py

```diff
@@ -36,11 +36,13 @@ def write_to_gcs(gcs_path, file_contents):
         os.write(fd, file_contents)
 
         command = ['gsutil', 'cp', f_name, gcs_path]
+        output = ''
         try:
             output = subprocess.check_output(command)
         except subprocess.CalledProcessError as e:
-            logging.error('Error encountered when writing to GCS!: {0}'
-                          .format(output))
+            logging.error('Error encountered when writing to GCS!')
+            if output is not '':
```

Comment:
Actually I don't think there's any chance output will be filled in here if an exception gets thrown, so you can probably just drop this.

To get the output in the error case, you have to get it off the exception object:
https://docs.python.org/2/library/subprocess.html#subprocess.check_output